style.css

```diff
@@ -1,554 +1,1111 @@
-}
-
-.mainhead > span {
-  display: block;
+  line-height: 4rem;
+  width: 100%;
+  height: 124px;
+  color: #ff6b00;
+  text-transform: capitalize;
+  font-family: 'Crete Round', serif;
 }
 
-.headline > h2 {
+.intro-container h2 {
+  color: #fff5e1;
   font-family: 'Roboto', sans-serif;
   font-weight: 700;
-  color: var(--txtcolor);
-  padding-top: 1rem;
+  font-size: 24px;
+  line-height: 2rem;
+  height: 27px;
+  width: 100%;
+  margin-top: 3%;
 }
 
-.headline > p {
-  color: var(--txtcolor);
-  padding-top: 1rem;
+.intro-container p {
   font-family: 'Poppins', sans-serif;
   font-style: normal;
-  font-weight: 300;
+  font-weight: 400;
   font-size: 16px;
-  line-height: 24px;
-  width: 20rem;
-  flex-grow: 0;
-  margin: 0 0;
-}
-
-.footericon > ul {
-  list-style-type: none;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-around;
-  padding-top: 2rem;
-  padding-bottom: 0.5rem;
-  padding-left: 0.5rem;
-  width: 70%;
+  line-height: 1.5rem;
+  height: 123px;
+  width: 82%;
+  margin-top: 4%;
+  color: #fff5e1;
 }
 
-.headline > ul {
-  list-style-type: none;
+section.intro-container ul {
+  height: 30px;
+  width: 60%;
+  margin-top: 15%;
   display: flex;
-  flex-direction: row;
-  justify-content: space-around;
-  width: 10rem;
-  padding-top: 2rem;
-  padding-bottom: 3.5rem;
-  padding-left: 0.5rem;
 }
 
-.headline > ul > li {
-  align-items: center;
-}
-
-/* work selector */
-.work {
-  display: grid;
-  justify-content: center;
-  align-content: center;
-  grid-auto-flow: row;
-  height: auto;
-  background: var(--txtcolor);
-  color: var(--txtcolor2);
+.intro-container li {
+  flex: 1;
+  list-style: none;
+  padding-top: 2%;
 }
 
-.about > h2 {
-  font-family: 'Crete Round', serif;
-  font-size: 4rem;
-  font-weight: 400;
-  line-height: 88px;
-  letter-spacing: 0.3700000047683716px;
-  text-align: left;
-  color: var(--maincolor);
+/* styling the recent work container */
+footer > hr {
+  background-color: #0d151e;
+  width: 35%;
+  height: 5px;
+  margin-left: 32%;
 }
 
-.contact > h2 {
-  font-family: 'Inter', sans-serif;
+.recent-work {
+  height: 112vh;
+  width: 100%;
+  background-image: none;
+  padding: 20% 24px;
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  text-align: center;
-  height: 10rem;
-  padding-top: 4rem;
-  width: 21.6rem;
-  left: 0;
-  top: 0;
-  color: var(--txtcolor2);
+  background-color: #fff;
 }
 
-.headtext {
-  margin: 0;
-  padding: 0;
-  border: none;
-}
-
-.work > .first-box > .headtext > h2 {
-  font-size: 2.2rem;
+.recent-work h2 {
+  font-size: 40px;
+  font-weight: 400;
   font-family: 'Crete Round', serif;
-  font-weight: 500;
-  justify-items: center;
-  padding: 5rem 1rem 2rem 0;
-  border-bottom: 1.5px solid var(--line);
-  width: 18rem;
-}
-
-.ttwork {
-  padding: 0%;
-  margin: 0%;
+  width: 100%;
+  line-height: 2rem;
+  height: 83px;
+  text-align: center;
+  margin-bottom: 10px;
+  color: #1c1a19;
```

Comment:
- [ ] Kindly correct the section inside the red lines to be similar to the Figma template. 👇🏽 
- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.
![image](https://user-images.githubusercontent.com/58642949/187038562-23043847-08c1-4a46-ad9c-3508cc4425a9.png)